src/Node/ModuleNode.php

```diff
@@ -115,7 +115,7 @@ protected function compileGetParent(Compiler $compiler)
         $parent = $this->getNode('parent');
 
         $compiler
-            ->write("protected function doGetParent(array \$context)\n", "{\n")
+            ->write("protected function doGetParent(array \$context): bool | string | Template | TemplateWrapper\n", "{\n")
```

Comment:
I think we don't add whitespace between the pipes (in Symfony I mean)

```suggestion
            ->write("protected function doGetParent(array \$context): bool|string|Template|TemplateWrapper\n", "{\n")
```